book.rb

```diff
@@ -0,0 +1,11 @@
+# Book class
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] You are missing the method for adding rentals. Kindly do that.

  You can import the **Rental** class here. Then set up an `add_rental` method that takes in `person` and `date` parameters and return an instance of the Rental object like so:
  ```
  def add_rental(date, person)
    Rental.new(date, person, self)     #=> notice the order of parameters
  end
  ```

Reply:
This issue is well resolved! ✔️